frontend/src/pages/accounts/details/StorageBadge.tsx

```diff
@@ -0,0 +1,14 @@
+import classes from "./StorageBadge.module.scss";
+
+type StorageBadgeProps = {
+  text: string;
+};
+
+export function StorageBadge({ text }: StorageBadgeProps) {
+  return (
+    <div className={classes.root}>
+      <div className={classes.badge}>{text}</div>
```

Comment:
Nit: What I'm about to describe really isn't that important in this case, but could be of greater importance in more complex components/functions.

Given that this component is named `StorageBadge` (and not just`Badge`), implies that it's adapted for the specific use-case - using it to show something related to storage. 

I can see that it's showing the "path domain" in this case - according to the usage:
https://github.com/onflowser/flowser/blob/447d45a03b4a83ac54b2fe72317de899f8c12e57/frontend/src/pages/accounts/details/StorageCard.tsx#L18-L20

If you really want to use it only for showing the path domain text, you could enforce that by specifying that it only accepts the `AccountStorageDomain` (path domain) type and maybe even change the name to something more specific, like so:

<img width="709" alt="Screenshot 2022-09-04 at 18 33 01" src="https://user-images.githubusercontent.com/36109955/188323710-a861df1e-5f3f-4f0b-986e-faff7230a66c.png">

This would make it clear that this component has a single responsibility of displaying the storage path domain. Otherwise other developers could soon start adapting that component to other specific use cases, so that component would then have multiple (different) "responsibilities".

This is a general software design concept, known as the "Single Responsibility Principle": https://en.wikipedia.org/wiki/Single-responsibility_principle


But if you want that `StorageBadge` component to be more generic and possibly used in other use-cases aswell, it's better to just the name to something more generic, like "Badge" or "OvalBadge".

Again this is really nitty and really not that important, but just throwing it out.